### PR TITLE
Latex table to mimic table from original book

### PR DESCRIPTION
**Pull request by DarthGandalf**
Part of #235 limit do trade table for now, and some common commands. I may need to extend these commands for other tables later

both-39

### CODE REVIEW

**Comment by qwrtln** [comment]
It would be useful to rip the original image as a background.

**Comment by DarthGandalf** [comment]
Can you do it?

File: sections/trading.tex [hunk 1]
```diff
@@ -20,8 +20,20 @@
...
+
+\note[0.95]{14}{
+	\bgroup
+	\setlength{\tabcolsep}{0.3em}
+	\centering
+	Trade Table
+	\bigskip
+
+	\begin{tabularx}{\linewidth}{XXXX}
+		\darkcell{\color{white}Sells/gets} & \darkcell{...to purchase \includesvg[height=10px]{\svgs/gold.svg}} & \darkcell{...to purchase \includesvg[height=10px]{\svgs/valuablegreater.svg}} & \darkcell{...to purchase \includesvg[height=10px]{\svgs/building_materials.svg}} \\
```
Comment (qwrtln) [inline sections/trading.tex:32]:
```suggestion
    \darkcell{\color{white}Selling\textbackslash{}to purchase} & \darkcell{...to purchase \includesvg[height=10px]{\svgs/gold.svg}} & \darkcell{...to purchase \includesvg[height=10px]{\svgs/valuablegreater.svg}} & \darkcell{...to purchase \includesvg[height=10px]{\svgs/building_materials.svg}} \\
```
Why linter doesn't pick up tabs is beyond me.

File: sections/trading.tex [hunk 2]
```diff
@@ -20,8 +20,20 @@
...
-\begin{figure*}[!hb]
-  \centering
-  \includegraphics[width=\linewidth]{\tables/trade_table.png}
-\end{figure*}
+
+\note[0.95]{14}{
+	\bgroup
+	\setlength{\tabcolsep}{0.3em}
+	\centering
+	Trade Table
```
Comment (qwrtln) [inline sections/trading.tex:28]:
```suggestion
	\textbf{Trade Table}
```

**Comment by DarthGandalf** [comment]
welp, the "note" background I used isn't big enough for the optional rules table

both-38

**Comment by qwrtln** [comment]
Here they are:

img-3771
img-3550
img-3386

**Comment by DarthGandalf** [comment]
Updated. It's still too small though

**Comment by DarthGandalf** [comment]
I've increased the size of background, looks better now

both-38

**Comment by qwrtln** [comment]
Much better. Any chance to use more subtle frame, like the one for `framedimage` instead of `note`? This thick frame is just ugly with the tables.

File: sections/difficulty.tex [hunk 3]
```diff
@@ -26,8 +26,38 @@
...
+  \small
+  {
+    \color{white}
+    You may modify the rules to increase or decrease the game's difficulty.
+  }
+  \bigskip
+
+  \begin{tabularx}{\linewidth}{>{\hsize=0.3\hsize\linewidth=\hsize}X
+                               >{\hsize=1.7\hsize\linewidth=\hsize}X}
+     \darkcell[1.5]{\color{white}Game Difficulty Levels} & \darkcell[1.5]{Change to the default rules} \\
```
Comment (qwrtln) [inline sections/difficulty.tex:43]:
```suggestion
     \darkcell[1.5]{\color{white}Game Difficulty Levels} & \darkcell[1.5]{\textbf{Change to the default rules}} \\
```

Comment (DarthGandalf) [reply sections/difficulty.tex:43]: I'll instead put it into \darkcell itself

**Comment by DarthGandalf** [comment]
both-27

**Comment by DarthGandalf** [comment]
both-51

File: sections/units.tex [hunk 4]
```diff
@@ -133,10 +133,26 @@ \subsection*{Neutral Units}
...
 \end{multicols}
 
-\begin{figure}[h]
-  \centering
-  \includegraphics[width=\linewidth]{\tables/unit_list.png}
-\end{figure}
+\hommtable{14}{
+  \begin{tabularx}{\linewidth}{XXXXX}
+    & \darkcell{Town Building} & \darkcell{Castle} & \darkcell{Dungeon} & \darkcell{Necropolis} \\
+    \darkcell[1.6]{\includesvg[height=12px]{\svgs/bronze.svg} \\ Bronze Units} &
```
Comment (qwrtln) [inline sections/units.tex:139]:
```suggestion
    \darkcell[1.6]{\includesvg[height=12px]{\svgs/bronze-table.svg} \\ Bronze Units} &
```

File: sections/difficulty.tex [hunk 5]
```diff
@@ -26,8 +26,38 @@
 
 \clearpage
 
-\includegraphics[width=\linewidth]{\tables/optional_rules.png}
-
-\vspace*{\fill}
+\vspace*{-2em}
+\hommtable{37}{
```
Comment (qwrtln) [inline sections/difficulty.tex:30]:
```suggestion
\hommtable{38}{
```

**Review by qwrtln — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**